packages/paste-core/components/toast/src/types.ts

```diff
@@ -0,0 +1,42 @@
+import * as PropTypes from 'prop-types';
+import {ValueOf} from '@twilio-paste/types';
+
+export const ToastIconVariants = {
+  ERROR: 'error',
+  NEUTRAL: 'neutral',
+  SUCCESS: 'success',
+  WARNING: 'warning',
+} as const;
+
+export const ToastRoles = {
+  ERROR: 'alert',
+  NEUTRAL: 'status',
+  SUCCESS: 'status',
+  WARNING: 'alert',
+} as const;
+
+export const ToastVariants = {
```

Comment:
Can this live in a constants file. It's not really a type.

Reply:
Done.